test/lint/lint-cppcheck-dash.py

```diff
@@ -130,7 +118,9 @@ def main():
 
     unique_sorted_lines = sorted(set(dependencies_output.stdout.splitlines()))
     for line in unique_sorted_lines:
-        if re.search(enabled_regexp, line) and not re.search(ignored_regexp, line) and re.search(files_regexp, line):
+        if not re.search(files_regexp, line):
+            continue
+        if re.search(enabled_regexp, line) or not re.search(ignored_regexp, line):
```

Comment:
so what's the point of having `ENABLED_CHECKS` then?

Reply:
also "if it's enabled then we don't care if it's ignored" logic feels wrong

Reply:
Let's maybe rename accordingly then e.g. `enabled` -> `important` or `enforced` or whatever. @coderabbitai can you suggest a better name for `ENABLED_CHECKS` considering the way it's used now?

Reply:
@coderabbitai any recommendation for "ignored checks" or is it good enough?